pandablocks/commands.py

```diff
@@ -283,6 +293,23 @@ class GetPcapBitsLabels(Command):
         GetPcapBitsLabels() -> PcapBitsLabels()
     """
 
+    def execute(self) -> ExchangeGenerator[Dict[str, List[str]]]:
+        ex = Exchange("PCAP.*?\n")
+        yield ex
+        bits_fields = []
+        for line in ex.multiline:
+            split = line.split()
+            if len(split) == 4:
+                field_name, _, field_type, field_subtype = split
+                if field_type == "ext_out" and field_subtype == "bits":
+                    bits_fields.append("PCAP.%s" % field_name)
+
+        result = map(lambda field: field + ".BITS?\n", bits_fields)
+        ex = Exchange(list(result))
+        yield ex
```

Comment:
Almost. An Exchange is a single command string to send, and it will get a single response. If we want multiple responses we need to yield a list of Exchanges which will be filled in parallel
```suggestion
        exchanges = [Exchange(f"{field}.BITS?") for field in bits_fields)]
        yield exchanges
```